README.md

```diff
@@ -68,4 +68,16 @@ Disable authentication when running locally by creating the configuration file `
 security.cas.bypass=true
 security.cas.disableCAS=true
 ```
+RequireApiKey annotation has been used in ShapeController.
 
+It supports standard 'apiKey' stored in http header.
+It also supports 'api_key' stored in http request params and POST body.
+
+Make sure ApiKey check url has been set in config:
+
+```
+security:
+    apikey:
+        check:
+            serviceUrl: "https://auth.ala.org.au/apikey/ws/check?apikey="
```

Comment:
This duplication is not required. Use https://github.com/AtlasOfLivingAustralia/spatial-service/blob/master/grails-app/conf/application.yml#L210

Rename existing property or copy it into this location required by the plugin. e.g. https://github.com/AtlasOfLivingAustralia/spatial-service/blob/master/grails-app/init/au/org/ala/spatial/service/BootStrap.groovy#L114

Reply:
Removed